### PR TITLE
Spellcheck of docs, a few minor changes

### PR DESCRIPTION
**Pull request by tommyod**
This PR continues my read-through of the docs, the previous PRs submitted are #18941 and  #18948.

The following edits have been made:
- Missing periods and colons added before introducing code examples.
- Increased number of function references (clickable links).
- Cleared up a few sentences which I found unclear.

Feedback is welcome.

### CODE REVIEW

File: doc/source/advanced.rst [hunk 1]
```diff
@@ -315,7 +316,7 @@ Basic multi-index slicing using slices, lists, and labels.
 
    dfmi.loc[(slice('A1','A3'), slice(None), ['C1', 'C3']), :]
 
-You can use a ``pd.IndexSlice`` to have a more natural syntax using ``:`` rather than using ``slice(None)``
+You can use ``pd.IndexSlice`` to facilitate a more natural syntax using ``:``, rather than using ``slice(None)``.
```
Comment (TomAugspurger) [inline doc/source/advanced.rst]: You could link to `IndexSlice` here

    :class:`IndexSlice`

I think (maybe `pandas.IndexSlice`).

File: doc/source/advanced.rst [hunk 2]
```diff
@@ -751,30 +756,31 @@ Scalar selection for ``[],.loc`` will always be label based. An integer will mat
...
 
 .. ipython:: python
 
    sf.iloc[3]
 
-A scalar index that is not found will raise ``KeyError``
+A scalar index that is not found will raise a ``KeyError``.
 
-Slicing is ALWAYS on the values of the index, for ``[],ix,loc`` and ALWAYS positional with ``iloc``
+Slicing is **always** on the values of the index when using ``[],ix,loc``, and
```
Comment (TomAugspurger) [inline doc/source/advanced.rst]: Except for booleans :)

Perhaps we can rephrase this as "**primarily** on the values" and mention elsewhere (perhaps after `iloc`) that booleans are also allowed.

File: doc/source/computation.rst [hunk 3]
```diff
@@ -47,17 +48,18 @@ NA/null values *before* computing the percent change).
 Covariance
 ~~~~~~~~~~
 
-The ``Series`` object has a method ``cov`` to compute covariance between series
-(excluding NA/null values).
+The ``Series`` object has a method :meth:`~Series.cov` to compute covariance
```
Comment (TomAugspurger) [inline doc/source/computation.rst]: How about

    :meth:`Series.cov` can be used to compute covariance between series (excluding missing values)

File: doc/source/computation.rst [hunk 4]
```diff
@@ -47,17 +48,18 @@ NA/null values *before* computing the percent change).
...
 
 .. ipython:: python
 
    s1 = pd.Series(np.random.randn(1000))
    s2 = pd.Series(np.random.randn(1000))
    s1.cov(s2)
 
-Analogously, ``DataFrame`` has a method ``cov`` to compute pairwise covariances
-among the series in the DataFrame, also excluding NA/null values.
+Analogously, ``DataFrame`` has a method :meth:`~DataFrame.cov` to compute
```
Comment (TomAugspurger) [inline doc/source/computation.rst]: And 

    Analogously, :meth:`DataFrame.cov` can be used to compute pairwise...

**Review by TomAugspurger — COMMENTED** [review]
Only partway through, looks good so far.

Style question: we have quite a few places like

    The :meth:`Series.cov` method does...

I prefer just saying

    :meth:`Series.cov` does...

I think it's clear from the context that we're talking about a method.

**Comment by tommyod** [comment]
Thanks for the feedback @TomAugspurger. I have submitted a new PR addressing your comments, which were helpful. 

- I agree that "*... to compute it, use `my_method` and ...*" is more succinct and clear than "*... to compute it, use the method `my_method` and ...*".
- I am uncertain about starting a sentence with "*The `my_method` method is...*" vs. "*`my_method` is...*". The first phrasing seems to occur quite a bit in the documentation.

File: doc/source/computation.rst [hunk 5]
```diff
@@ -110,6 +114,11 @@ Several methods for computing correlations are provided:
 .. \rho = \cov(x, y) / \sigma_x \sigma_y
 
 All of these are currently computed using pairwise complete observations.
+Wikipedia has articles covering the
```
Comment (jreback) [inline doc/source/computation.rst]: can you make these a bulleted list?

File: doc/source/indexing.rst [hunk 6]
```diff
@@ -941,9 +943,9 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 Indexing with isin
 ------------------
 
-Consider the ``isin`` method of Series, which returns a boolean vector that is
-true wherever the Series elements exist in the passed list. This allows you to
-select rows where one or more columns have values you want:
+Consider the :meth:`~Series.isin` method of Series, which returns a boolean 
+vector that is true wherever the Series elements exist in the passed list.
```
Comment (jreback) [inline doc/source/indexing.rst]: generally like to use double-backticks around Series, DataFrame, Index where possible.

File: doc/source/indexing.rst [hunk 7]
```diff
@@ -1123,8 +1125,6 @@ as condition and ``other`` argument.
                        'C': [7, 8, 9]})
    df3.where(lambda x: x > 4, lambda x: x + 10)
 
-**mask**
```
Comment (jreback) [inline doc/source/indexing.rst]: could make mask a sub-section (of the where section).

File: doc/source/indexing.rst [hunk 8]
```diff
@@ -1123,8 +1125,6 @@ as condition and ``other`` argument.
...
-
 ``mask`` is the inverse boolean operation of ``where``.
```
Comment (jreback) [inline doc/source/indexing.rst]: you can remove the experimental from the Query section.

File: doc/source/options.rst [hunk 9]
```diff
@@ -37,7 +37,7 @@ namespace:
 - :func:`~pandas.option_context` - execute a codeblock with a set of options
   that revert to prior settings after execution.
 
-**Note:** developers can check out pandas/core/config.py for more info.
+**Note:** Developers can check out ``pandas/core/config.py`` for more info.
```
Comment (jreback) [inline doc/source/options.rst]: you can put this link in: https://github.com/pandas-dev/pandas/blob/master/pandas/core/config.py

File: doc/source/options.rst [hunk 10]
```diff
@@ -79,15 +79,15 @@ Getting and Setting Options
 ---------------------------
 
 As described above, ``get_option()`` and ``set_option()`` are available from the
```
Comment (jreback) [inline doc/source/options.rst]: you can use ``:func:`get_option` `` and so on

File: doc/source/text.rst [hunk 11]
```diff
@@ -99,7 +99,7 @@ Elements in the split lists can be accessed using ``get`` or ``[]`` notation:
    s2.str.split('_').str.get(1)
    s2.str.split('_').str[1]
 
-Easy to expand this to return a DataFrame using ``expand``.
+It's easy to expand this to return a DataFrame using ``expand``.
```
Comment (jreback) [inline doc/source/text.rst]: It is

**Review by jreback — CHANGES_REQUESTED** [review]
lgtm! just some small changes. ping on green.

**Comment by tommyod** [comment]
@jreback I've addressed your comments. I've been noting some stylistic preferences (which I agree with), such as "*It is ...*" instead of "*It's*" and using double-backticks on objects. For now I have not addressed these globally in the docs, but I hope to do so in a future PR.

**Comment by jreback** [comment]
@tommyod totally cool. updating things like this is generally, do a few docs / files at once. it becomes too hard to review otherwise.

File: doc/source/indexing.rst [hunk 12]
```diff
@@ -1123,9 +1125,11 @@ as condition and ``other`` argument.
                        'C': [7, 8, 9]})
    df3.where(lambda x: x > 4, lambda x: x + 10)
 
-**mask**
 
-``mask`` is the inverse boolean operation of ``where``.
+Mask
+~~~~~~~~~
```
Comment (jreback) [inline doc/source/indexing.rst]: this needs to be the same length as the title (otherwise sphinx warns)

File: doc/source/indexing.rst [hunk 13]
```diff
@@ -1134,7 +1138,7 @@ as condition and ``other`` argument.
 
 .. _indexing.query:
 
-The :meth:`~pandas.DataFrame.query` Method (Experimental)
+The :meth:`~pandas.DataFrame.query` Method
```
Comment (jreback) [inline doc/source/indexing.rst]: same here

File: doc/source/indexing.rst [hunk 14]
```diff
@@ -1662,9 +1666,9 @@ Set an index
 
 .. _indexing.set_index:
 
-DataFrame has a ``set_index`` method which takes a column name (for a regular
-``Index``) or a list of column names (for a ``MultiIndex``), to create a new,
-indexed DataFrame:
+DataFrame has a :meth:`~DataFrame.set_index` method which takes a column name 
+(for a regular ``Index``) or a list of column names (for a ``MultiIndex``), 
+to create a new, indexed DataFrame:
```
Comment (jreback) [inline doc/source/indexing.rst]: capitalize as beginning of sentence?

File: doc/source/indexing.rst [hunk 15]
```diff
@@ -1703,9 +1707,10 @@ the index in-place (without creating a new object):
 Reset the index
 ~~~~~~~~~~~~~~~
 
-As a convenience, there is a new function on DataFrame called ``reset_index``
-which transfers the index values into the DataFrame's columns and sets a simple
-integer index. This is the inverse operation to ``set_index``
+As a convenience, there is a new function on DataFrame called 
+:meth:`~DataFrame.reset_index` which transfers the index values into the 
+DataFrame's columns and sets a simple integer index. 
+This is the inverse operation to ``set_index``.
```
Comment (jreback) [inline doc/source/indexing.rst]: put a ``:func:`DataFrame.set_index` ``

File: doc/source/indexing.rst [hunk 16]
```diff
@@ -1728,7 +1733,7 @@ discards the index, instead of putting index values in the DataFrame's columns.
 
 .. note::
 
-   The ``reset_index`` method used to be called ``delevel`` which is now
+   The ``reset_index`` method used to be called ``delevel``, which is now
```
Comment (jreback) [inline doc/source/indexing.rst]: you can remove this, delevel is removed

**Review by jreback — CHANGES_REQUESTED** [review]
small comments.

**Comment by tommyod** [comment]
@jreback :+1: . Addressed comments.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @tommyod
